client/components/jetpack/card/jetpack-product-card/display-price/paid.tsx

```diff
@@ -19,93 +20,117 @@ type OwnProps = {
 	expiryDate?: Moment;
 };
 
-const Paid: React.FC< OwnProps > = ( {
-	discountedPrice,
-	discountedPriceDuration,
+const Placeholder: React.FC< OwnProps > = ( { billingTerm, expiryDate } ) => {
+	return (
+		<>
+			<PlanPrice
+				original
+				className="display-price__original-price"
+				rawPrice={ 0.01 }
+				currencyCode="$"
+			/>
+			{ /* Remove this secondary <PlanPrice/> placeholder if we're not showing discounted prices */ }
+			<PlanPrice discounted rawPrice={ 0.01 } currencyCode="$" />
+			<TimeFrame expiryDate={ expiryDate } billingTerm={ billingTerm } />
```

Comment:
This code here results in lots of warnings in the console. Although the code was not added in this PR, it was rather added in this PR https://github.com/Automattic/wp-calypso/pull/62359/files#diff-339d3af13861616a5f855c40c24a389cdf976429f3a6f7d19356babe894c07c5

`currencyCode` here should be `USD`, instead of `$`.

<img width="931" alt="Screenshot 2022-12-29 at 3 14 13 PM" src="https://user-images.githubusercontent.com/18226415/209934078-c8927dcc-34a3-4ed6-bf02-bc20c830a766.png">